docs/source/software/systemwide/matlab.md

```diff
@@ -62,94 +62,379 @@ Use the following procedure if you need to install a MATLAB toolbox:
+>> job = c.batch(@pwd, 1, {}, 'CurrentFolder','.');
+
+>> % Query job for state
+>> job.State
+
+>> % If state is finished, fetch the results
+>> job.fetchOutputs{:}
+
+>> % Delete the job after results are no longer needed
+>> job.delete
+```
+
+To see the jobs that have been completed or still are running, you can call `parcluster` to return the cluster object that stores this information. Run the following command in the Command Window:
+
+```{code-block} matlab
+>> c = parcluster;
+>> jobs = c.Jobs
+>>
+>> % Get a handle to the second job in the list
+>> job2 = c.Jobs(2);
+```
+
+Once you locate the job, you can retrieve the results with `fetchOutputs` if you are using it interactively in the Command Window or you can use `load` if you are using it in a MATLAB script.
+
+```{code-block} matlab
+>> % Fetch all results from the second job in the list
+>> job2.fetchOutputs{:}
+```
+
+### Parallel MATLAB Batch Job
+The `batch` command can also submit parallel workflows to the HPC cluster. As an example, save the following code to a MATLAB script called `parallel_example.m.`
+
+```{code-block} matlab
+function [sim_t, A] = parallel_example(iter)
+ 
+if nargin==0
+    iter = 8;
 end
+ 
+disp('Start sim')
+ 
+t0 = tic;
+parfor idx = 1:iter
+    A(idx) = idx;
+    pause(2)
+    idx
+end
+sim_t = toc(t0);
+ 
+disp('Sim completed')
+ 
+save RESULTS A
+ 
+end
+```
+
+When you submit the job using the `batch` command, you will need to also specify the MATLAB Pool argument:
+
+```{code-block} matlab
+>> % Get a handle to the cluster
+>> c = parcluster;
+
+>> % Submit a batch pool job using 4 workers for 16 simulations
+>> job = c.batch(@parallel_example, 1, {16}, 'Pool',4, ...
+       'CurrentFolder','.');
+
+>> % View current job status
+>> job.State
+
+>> % Fetch the results after a finished state is retrieved
+>> job.fetchOutputs{:}
+ans = 
+   8.8872
+```
+
+This example took 8.89 seconds to run utilizing four workers. These jobs will always request N+1 number of CPUs for the task since 1 worker is required to manage the batch job and the pool of workers.
+
+We can run the same simulation but increase the size of the Pool. We will retrieve the results later so we will keep the submitted job id as a reference.
+
+```{note}
+Be careful with increasing the numbers of workers for your job because there can be a point where the returns on performance will diminish.
+```
+
+```{code-block} matlab 
+>> % Get a handle to the cluster
+>> c = parcluster;
+
+>> % Submit a batch pool job using 8 workers for 16 simulations
+>> job = c.batch(@parallel_example, 1, {16}, 'Pool',8, ...
+       'CurrentFolder','.');
+
+>> % Get the job ID
+>> id = job.ID
+id =
+   4
+>> % Clear job from workspace (as though MATLAB exited)
+>> clear job
 ```
 
-### Running the Job
-To run the myParallelAlgorithmFcn in MATLAB, in the MATLAB Command Window, type:
+You need to get a handle of the cluster and then you can use the `findJob` method to obtain information about the job:
```

Comment:
not sure what "get a handle of the cluster" means

Reply:
You need to get a handle of the cluster with the `parcluster` command ... 

I just added the command you need to use and it is the first step. I agree that it was awkwardly phrased but that should clear it up. Thank you!!